ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @author = author
+    @title = title
+    @rental = []
+  end
+end
```

Comment:
- [ ] You need to create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. Let's create a method for adding rentals like the example in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). It should be implemented similar to the following:

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

- Note:  The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
Adding it like this is giving the following error 
in `add_rental': uninitialized constant Book::Rental (NameError)

    Rental.new(person, self, date)